opencog/reasoning/pln/rules/inheritance-to-member.scm

```diff
@@ -0,0 +1,51 @@
+; =============================================================================
+; InheritanceToMemberRule
+;	InheritanceLink( B C )
+;			becomes
+;	MemberLink( B C )
+; -----------------------------------------------------------------------------
+
```

Comment:
Is it
```
InheritanceLink B C
|-
MemberLink B C
```
or
```
InheritanceLink {B} C
|-
MemberLink B C
```
?

As the code shows below, the former is implemented. IMO the only way to justify this is via using the intensional side of the inheritance, but clearly given the formula implemented that is not what is meant.